tools/barcode_collapse/BarcodeCollapse_pe.xml

```diff
@@ -0,0 +1,48 @@
+<tool id="barcode_collapse" name="barcode_collapse" version="0.1.0.0">
+    <description>Paired End randomer aware duplicate removal algorithm !</description>
+    <requirements>
+        <requirement type="package" version="0.1.0.0">barcode_collapse</requirement>
```

Comment:
I have seen that this tool depends on pysam, so you need to include: 
`<requirement type="package" version="0.8.4">pysam</requirement>`